packages/riipen-ui/src/components/Container.test.jsx

```diff
@@ -0,0 +1,104 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import defaultTheme from "../themes/default";
+
+import Container from "./Container";
+
+describe("<Container>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
```

Comment:
I think we should add a blank line between these two lines as this is the separating between our arrange and act steps of the test.

Reply:
Adjusted